docs/installation_guide.md

```diff
@@ -1,163 +1,238 @@
+  - Add improved / automated way to fetch rabbitmq password 
+  - Add description for PVC part
+  - 
+
 This document provides guidelines to installing all dependencies of DYNAMOS.
 
-**NOTE**: This document mainly focuses on Linux based commands. Thus we expect Windows users to enable Windows Subsystem for Linux 2. If there are any special instructions related to other operating systems, it will clearly be highlighted.
+**NOTE**: This document mainly focuses on Linux Debian based commands. Thus we expect Windows users to enable Windows Subsystem for Linux 2. If there are any special instructions related to other operating systems, it will clearly be highlighted.
+
+# Prerequisite software tools
+In order to setup DYNAMOS, we need to install a few tools as a foundation. This section covers such tools.
+
+## WSL2 (for Windows)
+Since we prefer to use Linux based operating systems, we suggest using the Windows Subsystem for Linux on a Windows machine. This will decrease the possibility of diverging issues when setting up the environment.
 
-# Prerequisites 
-## WSL2 (Windows)
 There are multiple ways to enable this, whether that be through the CLI, the Windows store or some other magical way. Here is one example of a tutorial:
+
 https://learn.microsoft.com/en-us/windows/wsl/install
+
 ## Docker Desktop
+For local development we utilise Docker Desktop, as upon installation we get Kubernetes for free in an easily manageable interface. 
+
+**NOTE**: In our development cycle we build and upload "finalized" images to docker hub, thus having a docker hub account may be useful if you intend to further develop services in DYNAMOS.
+
 https://www.docker.com/products/docker-desktop/
+
 Extra instructions: 
-1. Make sure to enable Kubernetes and provide sufficient resources to Docker when installed. We recommend at least 8GB of RAM and 4 CPU cores
-2. **For Windows users**: Make sure to enable WSL integration in the settings menu. If the option is not there, you may need to edit a script within windows to enable the integration.
-3. 
+1. Make sure to enable Kubernetes and provide sufficient resources to Docker when installed. We recommend at least 8GB of RAM and 4 CPU cores.
+2. **For Windows users**: Make sure to enable WSL integration in the settings menu. If the option is not there, you may need to edit a script within windows to enable the integration. If you are encountering issues with enabling WSL on Docker, use this blog as a reference: https://docs.docker.com/desktop/wsl/
+
+## Homebrew (optional)
+A package manager that may ease the setup process. This is automatically installed on most MacOS systems, however it requires some setup for Linux. 
+
+Homebrew for Linux:
+
+https://docs.brew.sh/Homebrew-on-Linux
+
+
+Homebrew is not available for Windows, however it is available for WSL. There are other package managers for Windows that might be useful, such as Chocolatey (https://chocolatey.org/), however we have not tested how useful it is here.
```

Comment:
The chocalatey part can be removed, since no Windows programs will be used at all.

Reply:
Removed